src/commands/admin/issue.ts

```diff
@@ -8,6 +8,7 @@ import { Message } from 'discord.js';
 export default class extends Command {
 
 	usage = '<title>|[options]';
+	description = `Creates an issue in ${BOT.NAME}' repository.`;
```

Comment:
```suggestion
	description = `Creates an issue in ${BOT.NAME}'s repository.`;
```
I've seen this file modified in both your and Blade's PRs, even though this is a Ren command. I've put this suggestion here just for good measure.